roles/docker/tasks/systemcontainer_crio.yml

```diff
@@ -125,6 +125,12 @@
     src: crio.conf.j2
     backup: yes
 
+- name: Configure the CNI network
```

Comment:
I think we need the following above this task. On a fresh install the directory doesn't exist.

```yaml
- name: Ensure /etc/cni/net.d/ exists
  file:
    path: /etc/cni/net.d/
    state: directory
    mode: 0755
```